cluster.go

```diff
@@ -61,6 +61,20 @@ func (c *ClusterClient) Close() error {
 	return nil
 }
 
+// PoolLen returns the number of active connections in the pool, for instrumentation
```

Comment:
What do you mean by "instrumentation"?

Also description is not very accurate since you return max number of active connections...